ClangAutoComplete.py

```diff
@@ -11,53 +11,53 @@ class ClangAutoComplete(sublime_plugin.EventListener):
 	compl_regex = re.compile("COMPLETION: ([^ ]+) : ([^\\n]+)")
 	file_ext = re.compile("[^\.]+\.([^\\n]+)")
 	syntax_regex = re.compile("\/([^\/]+)\.(?:tmLanguage|sublime-syntax)")
-	project_name_regex = re.compile("([^\.]+).sublime-project")
 	settings_time = 0
 
+	def to_bool(self, str_flag):
+		return str_flag == "true"
+
 	def load_settings(self):
 		# Only load the settings if they have changed
-		settings_modified_time = path.getmtime(sublime.packages_path()+"/ClangAutoComplete/"+"ClangAutoComplete.sublime-settings")
+		settings_modified_time = path.getmtime(
```

Comment:
Hey @pl-ca I actually think there is a better way to deal with it, I just don't know how yet. We probably could only load the settings once the plugin gets reloaded (it happens automatically on settings change, I guess) and then we can just keep the settings inside the class variables. This probably won't work quite as expected given we have now moved to having settings in a separate folder and not in the repo